<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulationWidgit</class>
 <widget class="QWidget" name="SimulationWidgit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>CrossCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(255,255,255,0);</string>
  </property>
  <widget class="QLabel" name="ScoreLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fixedsys</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Score:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ScoreAmountLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>22</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fixedsys</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="AmmoAmmountLabel">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>41</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fixedsys</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="AmmoPictureLabel">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>0</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="XLabel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>41</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fixedsys</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <zorder>ScoreLabel</zorder>
  <zorder>ScoreAmountLabel</zorder>
  <zorder>AmmoAmmountLabel</zorder>
  <zorder>XLabel</zorder>
  <zorder>AmmoPictureLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
